golos.referral/include/golos.referral/golos.referral.hpp

```diff
@@ -17,16 +17,16 @@ class referral: public contract {
     [[eosio::action]]
     void setparams(std::vector<referral_params> params);
 
-    [[eosio::action]]
-    void addreferral(name referrer, name referral, uint16_t percent, uint64_t expire, asset breakout);
+    [[eosio::action("addreferral")]]
+    void add_referral(name referrer, name referral, uint16_t percent, uint64_t expire, asset breakout);
 
     [[eosio::action]]
     void closeoldref();
 
     void on_transfer(name from, name to, asset quantity, std::string memo);
 
 private:
-    struct obj_referral {
+    struct addreferral {
```

Comment:
@afalaleev I renamed because in original ABI it is `addreferral`. It is simpler to make more changes identical for original than think will it cause some problems or not.